lib/hal/video.c

```diff
@@ -381,7 +381,7 @@ void XVideoInit(DWORD dwMode, int width, int height, int bpp)
 	_fb = framebufferMemory;
 
 	XVideoSetFlickerFilter(5);
-	XVideoSetSoftenFilter(TRUE);
+	XVideoSetSoftenFilter(FALSE);
```

Comment:
Why did you change it? I assume you found that retail titles do this?

Reply:
okay, I  booted Amped 1 and can confirm this is the case in traffic to the encoder.